README.md

```diff
@@ -1 +1,37 @@
-# hasek
\ No newline at end of file
+# hasek
+
+Important note:
+> All of the code here has been taken from [Giraffez](https://github.com/capitalone/giraffez)
+  So 100% of the credit should go to the team there!
+
+Hasek allows for easy password and secret management from either the command
+line or via Python code directly.
+
+
+## Quick Installation
+
+- A virtual environment is highly recommended!
+- and Python 3.6+
+
+```bash
+pip install git+https://github.com/fdosani/hasek.git
+```
+
+## Example
+
+First you can initialize the configuration with the following:
+```bash
+hasek config --init
```

Comment:
I remember thinking on a similar project that shall remain nameless that I could have just axed the `config` part here since it's not trying to do all the other fancy giraffez things.  Just do `hasek --init`, `hasek --set whatever`, etc... might be an idea here.  Save all those keystrokes :)